KEP 2535: Ensure Secret Pulled Images (#49886)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates/kubelet-ensure-secret-pulled-images.md

---
title: KubeletEnsureSecretPulledImages
content_type: feature_gate
_build:
  list: never
  render: false

stages:
  - stage: alpha
    defaultValue: false
    fromVersion: "1.33"
---
Forces the Kubelet to re-verify image pull credentials

> for pre-puled images.

Reviewer: A slightly more detailed description, please. Also, as I mentioned in an earlier comment - it's not just about pre-pulled images.

> Ensure that pods using an image are authorized to access the image
> with the provided credentials, even if the image is already present.

Reviewer: ```suggestion
Ensure that pods requesting an image are authorized to access the image
with the provided credentials when the image is already present on the node.
```
